README.md

```diff
@@ -46,12 +47,13 @@ Due to changes in CRD version support, the following versions of the chart are u
 
 ### CRDs Support of Traefik Proxy
 
-Due to changes in API Group of Traefik CRDs from `containo.us` to `traefik.io`, this Chart install the two CRDs API Group on the following versions:
+Due to changes in API Group of Traefik CRDs from `containo.us` to `traefik.io`, this Chart install CRDs needed by default Traefik Proxy version, following this table:
```

Comment:
Given that v3 is more or less a "fresh" start and all of the tables below are v2 related, we may consider cleaning it up entirely, listing only the minimum Helm/Kubernetes version and then add a single link referring to the `v27` `README` for those using v2? Then we can also drop the old upgrade notes from here.